modules/carto/src/api/layer-map.ts

```diff
@@ -144,8 +165,14 @@ export function getLayer(
   if (config.visConfig?.customMarkers) {
     basePropMap = mergePropMaps(sharedPropMap, customMarkersPropsMap);
   }
-  if (type === 'mvt' || type === 'tileset' || type === 'h3' || type === 'quadbin') {
-    return getTileLayer(dataset, basePropMap);
+  if (
```

Comment:
@zbigg @padawannn could you confirm that we will never get a layer type of `'mvt'` nor `'tileset'`? In this case we can remove them as they are not used in `LAYER_FROM_LAYER_TYPE`